### PR TITLE
Changes to make CMB in CPO easier

### PR DESCRIPTION
**Pull request by prestontw**
- Does not crash (tested with writing a basic program and running it, though using WeScheme parser)
- Makes CMB's constructor more like CodeMirror's, making migration easier.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -98,7 +98,7 @@
     "codemirror": "5.44.0"
   },
   "optionalDependencies": {
-    "wescheme-js": "bootstrapworld/wescheme-js"
+    "wescheme-js": "github:bootstrapworld/wescheme-js"
```
Comment (prestontw) [inline package.json]: Not sure if we want to include this change...

Comment (schanzer) [reply package.json]: What's the benefit of this change?

Comment (prestontw) [reply package.json]: It happened auto-magically after I installed `wescheme-js`.

Comment (schanzer) [reply package.json]: Huh. OK. Well, I'm good to merge this. Nice work!

**Pull request merged**